ftw/publisher/core/utils.py

```diff
@@ -334,6 +334,13 @@ def make_path_relative(path):
     return path[len(site_path + '/'):]
 
 
+def make_path_absolute(path):
+    """Make a relative path absolute by adding the site root.
+    """
+    site_path = '/'.join(getSite().getPhysicalPath())
+    return '/'.join([site_path, path])
```

Comment:
If a `path` would start with a slash, we would end up with two slashes.
I'd probably handle this here since it is easy to do and we can avoid problems.
```diff
-    return '/'.join([site_path, path]) 
+    return '/'.join([site_path, path.lstrip('/')]) 
```